lib/configure.js

```diff
@@ -1,4 +1,5 @@
 module.exports = exports = configure
+module.exports.test = { findPython: findPython }
```

Comment:
To make it clear that it's not part of the official API but that it's only exported for testing.